chall01/spentti.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import sys
+import re
+
+morse = {
+	'a': '.-',
+	'b': '-...',
+	'c': '-.-.',
+	'd': '-..',
+	'e': '.',
+	'f': '..-.',
+	'g': '--.',
+	'h': '....',
+	'i': '..',
+	'j': '.---',
+	'k': '-.-',
+	'l': '.-..',
+	'm': '--',
+	'n': '-.',
+	'o': '---',
+	'p': '.--.',
+	'q': '--.-',
+	'r': '.-.',
+	's': '...',
+	't': '-',
+	'u': '..-',
+	'v': '...-',
+	'w': '.--',
+	'x': '-..-',
+	'y': '-.--',
+	'z': '--..',
+	' ': ' '
+}
+
+if (len(sys.argv) == 2):
+	word = sys.argv[1].lower().strip()
+	mode = 0
+	for c in word:
+		if c.isalpha() or c == ' ':
+			mode = mode
+		else:
+			mode = 1
+	if (mode == 0 and len(word) > 0):
+		for char in word:
+			sys.stdout.write(morse[char])
```

Comment:
like in C, try to avoid as much as possible to use syscall. So you shouldn't print char by char but append morse[char] to a string and then print this string at the end.